bin/dfx-software-sns-aggregator-ci-wasm-url

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+set -euo pipefail
+SOURCE_DIR="$(dirname "$(realpath "${BASH_SOURCE[0]}")")"
+PATH="$SOURCE_DIR:$PATH"
+. "$SOURCE_DIR/versions.bash"
+
+print_help() {
+  cat <<-EOF
+
+	Outputs the URL of the wasm file for the sns_aggregator canister.
+
+	EOF
+}
+
+# Source the clap.bash file ---------------------------------------------------
+source "$SOURCE_DIR/clap.bash"
+# Define options
+clap.define short=r long=release desc="The release name" variable=DFX_RELEASE default="pinned"
```

Comment:
It's the aggregator release, not the DFX release, right? Or what is the `DFX_` prefix signifying here?

Reply:
My hope is that many of these scripts are polyfills and will eventually make their way into dfx core or dfx extensions.  The DFX_ prefix to inputs follows from that hope.    dfx has traditionally takes some inputs as flags, others as environment variables.  The dfx executable names its environment variables DFX_xxx.  If a polyfill is implemened in dfx, we can first call it directly in the polyfill, fixing up and differences as needed, then eventually eliminate the polyfill.  Hopefully the variable names will be compatible enough that the implementer will not be tempted to change them.

Reply:
That said, the name is confusing, I agree.  Something like `DFX_AGGREGATOR_RELEASE` would be better.

Reply:
Done.